.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
I don't think we should be committing editor specific stuff

Reply:
I've seem Typescript related config as the exception sometimes due to the miss-match in editor plugins. I'll remove for this iteration.